hw06-atm-emulator/src/main/java/ru/otus/Builders/IDispenserBuilder.java

```diff
@@ -0,0 +1,7 @@
+package ru.otus.Builders;
+
+import ru.otus.IDispenser;
+
+public interface IDispenserBuilder {
+    IDispenser Build();
```

Comment:
Имена методов с маленькой буквы.
Не только здесь - везде (я уже в других местах не пишу).

```suggestion
    IDispenser build();
```